website/content/docs/job-specification/network.mdx

```diff
@@ -105,6 +105,12 @@ All other operating systems use the `host` networking mode.
 - `host_network` `(string:nil)` - Designates the host network name to use when allocating
   the port. When port mapping the host port will only forward traffic to the matched host
   network address.
+- `ignore_collision` `(bool: false)` - Allows the group to be placed on a node
+  where the port may already be reserved. Intended for programs that support
```

Comment:
Do we want to use the word "reserved" here? There's a specific client configuration option for "reserved ports" that may be confusing. Maybe "allocated" or even just "in use" (maybe the port isn't being used by a Nomad job at all)?

Reply:
I actually imagine that this may be used in conjunction with the [`reserved_ports` client config](https://developer.hashicorp.com/nomad/docs/configuration/client#reserved_ports), so a cluster operator may dedicate ports for applications like these, and run them with Nomad instead of systemd or such.

On the flip side, it also allows job authors to bypass operator intent...  I suppose Sentinel may need to come into play here to retain current guarantees. :thinking:

Reply:
I thought some more about bypassing operator intent around `reserved_ports`: a job author can already do that today, if they merely do not specify a `port{}`, so this does not make that situation any worse.  (I have had to remind myself of this more than once.)

Reply:
Excellent point!